### PR TITLE
Include dryrb information in documentation

### PR DESCRIPTION
**Pull request by cdaloisio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,3 +173,40 @@ We satisfy the SAP by:
...
+```
+
+to this:
+
+```haskell
+do
+  x1 <- action1
+  x2 <- action2
+  mk_action3 x1 x2
+```
```
Comment (michaelwebb76) [inline README.md]: This is great, but I think if we consistently make addendums like this without a broader category/index it's going to end up not being as useful. I'm wondering if we should split the README into sections with Philosophy, Structure, Conventions, etc

Comment (cdaloisio) [reply README.md]: Valid feedback and I agree - I will whip it into shape :+1: I'll even add a "table of contents" :scream: :laughing:

**Comment by cdaloisio** [comment]
@michaelwebb76 I've made some very minor adjustments to headings and added a TOC - let me know what you think

**Comment by michaelwebb76** [comment]
👍

**Pull request merged**